torch/fx/node.py

```diff
@@ -439,6 +439,20 @@ def is_impure(self):
 
         return False
 
+    def replace_input_with(self, old_input: 'Node', new_input: 'Noe'):
+        """
+        Replace input node with new input node.
+        """
```

Comment:
Please format docstrings according to the [google docstring style guide](https://google.github.io/styleguide/pyguide.html), including a description, `Args`, `Return`, and ideally an example invocation